wvd-sh/WVD scaling script/Config.json

```diff
@@ -3,56 +3,52 @@
 		"Azure": {
 			"Variable": [
 				{
-					"_Comments":"Tenant ID of Azure Active Directory",
+					"_Comments": "Tenant ID of Azure Active Directory",
 					"Name": "AADTenantId",
 					"Value": "00000000-0000-0000-0000-000000000000"
 				},
-				{	"_Comments":"Stored Credentials of Azure Service Principal GUID",
+				{
+					"_Comments": "Stored Credentials of Azure Service Principal GUID",
 					"Name": "AADApplicationId",
 					"Value": "00000000-0000-0000-0000-000000000000"
 				},
 				{
-					"_Comments":"use AADServicePrincipalSecret filed for testing only, set via New-StoredCredential",
-					"Name": "AADServicePrincipalSecret",
-					"Value": ""
-				},
-				{
-					"_Comments":"Subscription Id of the Azure",
+					"_Comments": "Subscription Id of the Azure",
 					"Name": "currentAzureSubscriptionId",
 					"Value": "00000000-0000-0000-0000-000000000000"
 				},
 				{
-					"_Comments":"WVD tenant Group name",
+					"_Comments": "WVD tenant Group name",
 					"Name": "tenantGroupName",
 					"Value": "Default Tenant Group"
 				},
 				{
-					"_Comments":"WVD tenant name",
+					"_Comments": "Name of the WVD Tenant",
 					"Name": "tenantName",
 					"Value": ""
 				},
 				{
-					"_Comments":"WVD host pool name",
+					"_Comments": "Name of the WVD Host Pool",
 					"Name": "hostPoolName",
 					"Value": ""
 				},
 				{
-					"_Comments":"Hostpool session Load Balancing Type in Peak Hours",
+					"_Comments": "Hostpool session Load Balancing Type in Peak Hours",
 					"Name": "PeakLoadBalancingType",
 					"Value": "BreadthFirst"
 				},
 				{
-					"_Comments":"Hostpool session Load Balancing Type in OffPeak Hours",
+					"_Comments": "Hostpool session Load Balancing Type in OffPeak Hours",
 					"Name": "OffPeakLoadBalancingType",
 					"Value": "DepthFirst"
 				},
 				{
-					"_Comments":"Session Load Balancing Peak Hours",
+					"_Comments": "Session Load Balancing Peak Hours",
 					"Name": "sessionLoadBalancingPeakHours",
 					"Value": "9:00-18:00"
 				},
 				{
-					"_Comments":"Session Load Balancing off Peak Hours",
+					"_Comments": "Session Load Balancing off Peak Hours",
```

Comment:
This should not be a variable. There should be a variable for "On-Peak" (which you have above), and that's it. The remaining time is assumed to be off-peak.

Reply:
Removed off-peak input parameters
https://github.com/Azure/RDS-Templates/pull/171/commits/e4143629129d5bf734f9420c43dbe4947a409208